docs/modules/configuring/con-configuring-client-connections.adoc

```diff
@@ -0,0 +1,67 @@
+[id='con-configuring-client-connections-{context}']
+= Securing connections from clients
+
+[role="_abstract"]
+To secure client connections to virtual clusters, configure TLS on the virtual cluster by doing the following:
+
+Obtain a CA (Certificate Authority) certificate for the virtual cluster from a Certificate Authority.
```

Comment:
```suggestion
Obtain a server certificate for the virtual cluster from a Certificate Authority (CA).
```
"CA (Certificate Authority) certificate" makes it sound like Kroxlicious requires a CA's cert.  It does not.  It does not issue certificates itself.